quality-assurance/Security-Testing-Guidance.md

```diff
@@ -0,0 +1,116 @@
+# Overview
```

Comment:
On the filename - could it be "security-testing.md" (and in lower case) to be consistent with the current files.